packages/synapse-interface/constants/chains/master.tsx

```diff
@@ -181,7 +181,7 @@ export const POLYGON: Chain = {
   nativeCurrency: { name: 'Matic', symbol: 'MATIC', decimals: 18 },
   explorerUrl: 'https://polygonscan.com',
   explorerName: 'PolygonScan',
-  explorerImg: polygonExplorerImg,
+  explorerImg: polygonImg,
```

Comment:
We're using the SVG in `@assets/chains/...` here, meaning can we do that throughout so we don't have duplicates?

Reply:
Thanks for catching this! I was just testing out a few diff things to try and get the .png file to work. Didn't end up working so I converted all of `@assets/explorer/` files from .png to .svg

Switched back in https://github.com/synapsecns/sanguine/pull/1549/commits/dde17ade86d7d33a4d5af2e619fd7e2072d27c71